frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditorSidebar/NativeQueryEditorSidebar.module.css

```diff
@@ -1,17 +1,10 @@
-.Container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-
-.RunButtonWithTooltipStyled {
-  margin: 1rem;
-  margin-top: auto;
-  height: 40px;
-  width: 40px;
-}
-
 .SidebarButton {
-  padding: 0;
-  margin-top: 1rem;
+  color: var(--mb-color-text-primary);
+  transition: color 0.3s linear;
```

Comment:
Can we check with product on the transitions? I've always been under the impression that we're not super big on them. cc @kdoh